go.mod

```diff
@@ -1,6 +1,8 @@
 module zotregistry.io/zot
 
-go 1.20
+go 1.21
```

Comment:
Please leave the go 1.21 update for a separate PR. There are multiple changes to be made for this, which I have in: https://github.com/project-zot/zot/pull/2049

Reply:
I've removed the go.* file changes. But, I'm not able to build this with go 1.21, as it refuses to upgrade the dependencies due to:

```console
$ go mod tidy -go=1.20 
go: github.com/letsencrypt/boulder@v0.0.0-20231026200631-000cd05d5491 requires go@1.21, but 1.20 is requested
```

Reply:
BTW, I can wait for https://github.com/project-zot/zot/pull/2049 to land, and then rebase on master then.